lib/ControlledVocab/ControlledVocab.js

```diff
@@ -499,7 +543,11 @@ class ControlledVocab extends React.Component {
           <ConfirmationModal
             id="delete-controlled-vocab-entry-confirmation"
             open={this.state.showConfirmDialog}
-            heading={<FormattedMessage id="stripes-core.button.deleteEntry" values={{ entry: type }} />}
+            heading={
+              deleteEntry ?
+                <FormattedMessage id={deleteEntry} /> :
+                <FormattedMessage id="stripes-core.button.deleteEntry" values={{ entry: type }} />
+            }
```

Comment:
Can you add more tests to bump the code coverage of new code from 58.3% to at least 80%?
https://sonarcloud.io/component_measures?metric=new_coverage&selected=org.folio%3Astripes-smart-components%3Alib%2FControlledVocab%2FControlledVocab.js&view=list&pullRequest=1280&id=org.folio%3Astripes-smart-components

Reply:
@zburke The pull request looks promising and works for German.